db/migrate/20220629080906_add_note_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddNoteToOrders < ActiveRecord::Migration[6.1]
+  def change
+    add_column :spree_orders, :note, :string
```

Comment:
Shall we add `nil: false`? The empty string is just as good here, I guess.

Reply:
I don't see any change here. Is there a reason to keep `nil` values?

Reply:
No. 
Do you prefer empty string over null object? I've not opinion here.
Then, should we add `null: false, default: ""` ?

Reply:
I made the changes: 7fbe89392

Reply:
Thank you! My preference is to have the empty string to avoid nil errors. Queries can also be simpler with `where(note: "")` instead of checking two values.